doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -1,23 +1,31 @@
-This file contains a walkthrough of the NumPy 1.14.4 release on Linux.
+This file contains a walkthrough of the NumPy 1.14.5 release on Linux.
 The commands can be copied into the command line, but be sure to
-replace 1.14.4 by the correct version.
+replace 1.14.5 by the correct version.
 
 Release  Walkthrough
 ====================
 
+Backport Pull Requests, Write Release Note
+---------------------------------------------
+
+Changes that have been marked for this release must be backported to the
+maintenance/1.14.x branch. Summarize the changes in a release note
+doc/release/1.14.5-notes.rst
+
 Update Release documentation
 ----------------------------
 
-The file ``doc/changelog/1.14.4-changelog.rst`` should be updated to reflect
+The file ``doc/changelog/1.14.5-changelog.rst`` should be updated to reflect
 the final list of changes and contributors. This text can be generated by::
 
-    $ python tools/changelog.py $GITHUB v1.14.3..maintenance/1.14.x > doc/changelog/1.14.4-changelog.rst
+    $ python tools/changelog.py $GITHUB v1.14.4..maintenance/1.14.x > doc/changelog/1.14.5-changelog.rst
 
 where ``GITHUB`` contains your github access token. This text may also be
-appended to ``doc/release/1.14.4-notes.rst`` for release updates, though not
+appended to ``doc/release/1.14.5-notes.rst`` for release updates, though not
 for new releases like ``1.14.0``, as the changelogs for latter tend to be
 excessively long. The ``doc/source/release.rst`` file should also be
-updated with a link to the new release notes.
+updated with a link to the new release notes. These changes should be committed
+to the maintenance branch, I used prepare-1.14.5-release as a branch name.
```

Comment:
Is the branch name relevant? Also, I think you are saying that these changes should be added to both the maintenance and master branch, maybe say that explicity.

Reply:
Branch name removed. The forward port to the master branch is mentioned below in the post release tasks